package.json

```diff
@@ -48,14 +49,15 @@
     "grunt-newer": "^0.7.0",
     "grunt-sass": "^1.2.0",
     "highlight.js": "^8.0.0",
-    "karma": "^0.12.16",
-    "karma-chai-sinon": "^0.1.3",
+    "karma": "^1.3.0",
+    "karma-babel-preprocessor": "^6.0.1",
```

Comment:
We are. Tests are being run against the built dist output. Travis runs `npm run build`. There were issues with the older versions of karma not wanting to run against dist in Travis, hence the version bump.